hphp/runtime/vm/jit/vasm-lower-inl.h

```diff
@@ -38,14 +44,23 @@ void vasm_lower(Vunit& unit, Vlower lower_impl) {
   // iterators.
   auto& blocks = unit.blocks;
 
+  VLS env{unit, true};
```

Comment:
nit: Just `VLS env { unit };`, since the `true` is always going to be overridden before anything happens.